src/js/PhotoSphereViewer.js

```diff
@@ -157,7 +157,12 @@ function PhotoSphereViewer(options) {
       cropped_height: 0,
       cropped_x: 0,
       cropped_y: 0
-    }
+    },
+    usexmpdata: false, // most panoramas doesnt have xmpdata, and checking for them require to download the image twice (???)
```

Comment:
I've got panorama photos made with different smartphones, with ricoh theta and other cameras; never found xmp details.. 
Anyway, while using `usexmpdata` the image file get loaded twice